Client/play/src/CluesContainer.jsx

```diff
@@ -0,0 +1,130 @@
+'use strict';
+/* jshint quotmark: false */
+
+var React = require('react');
+var TitleBox = require('./TitleBox');
+var Clues = require('./Clues');
+var gMap = require('../../../lib/gMapLib');
+var Status = require('./Status');
+var PinSuccess = require('./PinSuccess');
+var HuntSuccess = require('./HuntSuccess');
+var Actions = require('../RefluxActions');
+
+var HITDISTANCE = 4;
+var windowHeight = { height: window.innerHeight }; 
+
+module.exports = React.createClass({
+
+  getInitialState: function() {
+
+    var currentPinIndex = this.props.hunt.currentPinIndex;
+    var numOfPins = this.props.hunt.pins.length;
+    currentPinIndex = Math.min(currentPinIndex, numOfPins-1);
+    var currentPin = this.props.hunt.pins[currentPinIndex];
+    var answer = currentPin.answer;
+    var resultText = currentPin.resultText;
+
+    return {
+      playerAtLocation: false,
+      huntComplete: false,
+      distanceToNextPin: 0.00,
+      answer: answer,
+      resultText: resultText
+
+    };
+  },
+  componentWillMount: function() {
+    var currentPinIndex = this.props.hunt.currentPinIndex;
+    var numOfPins = this.props.hunt.pins.length;
+    currentPinIndex = Math.min(currentPinIndex, numOfPins-1);
+    var currentPin = this.props.hunt.pins[currentPinIndex];
+    var nextGeo = currentPin.geo;
+    gMap.getDistanceToLatLng(function (value) {
+      var playerAtLocation = false;
+      var huntComplete = false;
+      if (value < HITDISTANCE) {
+        Actions.updateHuntAtKey(this.props.hunt.currentPinIndex + 1, 'currentPinIndex');
+        playerAtLocation = true;
+        value = 0;
+      }
+      if (playerAtLocation && this.props.hunt.currentPinIndex >= (this.props.hunt.pins.length - 1)) {
+        huntComplete = true;
+      }
+      this.setState({
+        playerAtLocation: playerAtLocation, 
+        distanceToNextPin: value,
+        huntComplete: huntComplete
+      });
+    }.bind(this), nextGeo);
+  },
+
+  togglePlayerAtLocation: function() {
```

Comment:
This should be called something else as it does not toggle. Perhaps clearPlayerAtLocation?